### PR TITLE
Adding gapter to documentation index for distro

### PR DESCRIPTION
**Pull request by Maram-Alajlan**

### CODE REVIEW

**Comment by tfoote** [comment]
These are not valid git repo urls. 
```
ERROR: In file '''/home/travis/build/ros/rosdistro/indigo/distribution.yaml''': Could not validate source entry for repo gapter with error [[[Could not validate repository with url https://github.com/aniskoubaa/gaitech_edu/tree/master/src/gapter and version master from entry at line '''3464''']]]
ERROR: In file '''/home/travis/build/ros/rosdistro/indigo/distribution.yaml''': Could not validate doc entry for repo gapter with error [[[Could not validate repository with url https://github.com/aniskoubaa/gaitech_edu/tree/master/src/gapter and version master from entry at line '''3455''']]]
```

Also unless you've run bloom and are making a release please skip the release section.

Please see the tutorial here: http://wiki.ros.org/rosdistro/Tutorials/Indexing%20Your%20ROS%20Repository%20for%20Documentation%20Generation

**Comment by Maram-Alajlan** [comment]
```  
======================================================================
ERROR: test.test_build_caches.test_build_caches
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/home/travis/build/ros/rosdistro/test/test_build_caches.py", line 50, in test_build_caches
    raise RuntimeError('\n'.join(errors))
RuntimeError: indigo: missing package.xml file for package "gapter"
```

How can I fix.

File: indigo/distribution.yaml [hunk 1]
```diff
@@ -3451,6 +3451,22 @@ repositories:
       url: https://github.com/fzi-forschungszentrum-informatik/fzi_icl_core.git
       version: master
     status: maintained
+  gapter:
+    doc:
+      type: hg
```
Comment (tfoote) [inline indigo/distribution.yaml]: This section is typically the same as the source section below.

Comment (Maram-Alajlan) [inline indigo/distribution.yaml]: What does it mean and what I should do?

Comment (tfoote) [inline indigo/distribution.yaml]: Copy the content of your source element below into the doc section here.

**Pull request merged**